### DISCUSSION TITLE
LARKMAIL NEEDS TRACKER

### DISCUSSION THREAD

**Discussion by observerSora** [go-lark/lark#49] [Q&A]
Hey!

Have you discovered an app or extension with Lark that can provide a real-time mail tracker inside the app? 
Just think of the feature given by MailTracker to Gmail. Giving notification or indicator that the email was already seen by the recipient.

Hope you can help me. Our company recently moved our whole company process and communications to Lark, that includes Larkmail with our lead client. 

Thank you.

**Comment by crispgm** [comment]
Hey @observerSora 

I am sorry that I have no idea with your question. Because go-lark is a third-party Lark messaging API SDK, I personally do not highly connect with Lark team and all of its functionality.

You may contact a sales person for more information :)
